packages/react-native/src/Authenticator/Defaults/VerifyUser/VerifyUser.tsx

```diff
@@ -82,7 +82,9 @@ const FormFields: DefaultVerifyUserComponent['FormFields'] = ({
         <Radio
           {...props}
           key={value}
-          value={value}
+          // value has to be name, because Auth is only interested in the
+          // string "email" or "phone_number", not the actual value
```

Comment:
Not sure if this comment is necessary, but thought it'd help later because this is fairly Amplify JS specific detail.